src/json_obj.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <unordered_map>
+#include <string>
+#include "json_value.h"
+
+class JsonObj {
```

Comment:
Just notice something quite important. The namespace is missing here. It will be needed as the projects expands. If others want to use this project, it also prevents conflicts with existing functions if there is already a JsonObj class.